runtime/calamari/src/lib.rs

```diff
@@ -484,7 +453,7 @@ parameter_types! {
 	// Pallet account for record rewards and give rewards to collator.
 	pub const PotId: PalletId = PalletId(*b"PotStake");
 	// How many collator candidates is allowed.
-	pub const MaxCandidates: u32 = 10;
+	pub const MaxCandidates: u32 = 50;
```

Comment:
Again why this value.

Reply:
We followed acala's configuration as well, it should be not too small or too big.
Definitely we can modify these constants by runtime upgrade.

Reply:
Great thanks. Just wanted to include the reasoning in the PR, if someone reads through it in the future.